// try {
//   throw "error1";
// } catch (e) {
//   console.log(e);
// }

console.log(!!1);
console.log(!!0);
console.log(!!-1);
